server/app/models/Applicant.java

```diff
@@ -51,6 +51,16 @@ public Applicant() {
     super();
   }
 
+  /**
+   * Clears applicantData so that a new ApplicantData object will be returned from getApplicantData.
+   * Useful when applicantData has been stored as an empty object and needs to be udpated with the
+   * data stored in object
+   */
+  public ApplicantData getApplicantData(boolean expireCache) {
```

Comment:
**suggestion (blocking):** Generally speaking, booleans should be avoided in method parameters because they:
- give no suggestion to what they're for at the call site without an inline comment
- need to be replaced with enums when more than two states become necessary

In this case I'd recommend removing the param, changing the method name to `getApplicantDataSkipCache` (or similar), removing `public` and adding the `@VisibleForTesting` annotation